manifests/base/config/metrics_allowlist.yaml

```diff
@@ -139,3 +140,9 @@ data:
         expr: sum(rate(apiserver_request_total{job=\"apiserver\"}[1h])) by(code, instance)
       - record: sum:apiserver_request_total:5m
         expr: sum(rate(apiserver_request_total{job=\"apiserver\"}[5m])) by(code, instance)
+      - record: rpc_rate:grpc_server_handled_total:sum_rate
+        expr: sum(rate(grpc_server_handled_total{job=\"$cluster\",grpc_type=\"unary\",grpc_code!=\"OK\"}[5m]))
+      - record: active_streams_watch:grpc_server_handled_total:sum
```

Comment:
this expr cannot work in recordingrule since it still has a variable job=\"$cluster\"